### ISSUE TITLE
fman doesn't start after screen resolution change

### ISSUE THREAD

**Issue by ave-63** [fman-community/fman#569]
fman was working fine a couple hours ago. Then I lowered my screen resolution to 1440 x 900. The font in fman became very large so I closed it, and it wouldn't reopen. 

There are a couple error-handling processes running. [Here is a screenshot.](https://i.imgur.com/ahT3fe7.png)

I thought maybe I could get an error message when I ran fman.exe in from a command prompt but nothing happened.

I'm on a Surface Pro 6, with Windows 10 up to date.

**Comment by mherrmann** [comment]
Could it be that it has been quarantined by Windows Defender? See #568.

**Comment by ave-63** [comment]
Well, it's working now... but I'm not sure it was the Windows Defender issue. I couldn't find any sign of a problem in Windows Defender or any mention of Tiggre!plock. I followed the instructions to update the definitions in command prompt. I restarted the computer and fman still wouldn't start. I re-installed it, just to see what would happen, and then it worked!

So maybe Windows Defender deleted something, without telling me about it. More like Windows Offender!
